core/core_bind.h

```diff
@@ -128,6 +128,7 @@ class OS : public Object {
 public:
 	enum RenderingDriver {
 		RENDERING_DRIVER_VULKAN,
+		RENDERING_DRIVER_D3D12,
```

Comment:
Probably it should be added after `RENDERING_DRIVER_OPENGL3` for compatibility.

Reply:
I wanted to keep the RD-ones grouped, but backcompat is more important indeed. Done!